ui/dist/index.html

```diff
@@ -8,8 +8,38 @@
     />
     <link rel="stylesheet" type="text/css" href="/index.css" />
     <link rel="icon" type="image/png" href="/img/logo-only.svg" />
+
+    {{ if .IncludeGoogleTagManager }}
```

Comment:
(totally optional)

I think you could simplify this to use just the `GoogleTagManagerID` data entry, if you change this line and line 30 to be `{{ with .GoogleTagManagerID }}`, and change lines 24 and 34 to use `{{ . }}`. (Up to you.)